src/api/java/com/minecolonies/api/colony/colonyEvents/descriptions/IBuildingEventDescription.java

```diff
@@ -0,0 +1,41 @@
+package com.minecolonies.api.colony.colonyEvents.descriptions;
+
+/**
+ * Event description for building events.
+ */
+public interface IBuildingEventDescription extends IColonyEventDescription
```

Comment:
Please rename the methods in this interface to be more accurate.
Aka `getBuildingName` instead of `getBuilding`

Reply:
done.